### PR TITLE
Create government.md

### PR DESCRIPTION
**Pull request by hrazi**

### CODE REVIEW

File: articles/communication-services/concepts/government.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Azure Communication Services for Government
+
+
+Azure Communication Services (ACS) in [Azure Government](https://azure.microsoft.com/en-us/global-infrastructure/government/) provides compliance with US government requirements for cloud services. In addition to enjoying the features and capabilities of Messaging, Voice and Video calling, developers benefit from the following features that are unique to Azure Government:
```
Comment (ProbablePrime) [inline articles/communication-services/concepts/government.md]:
```suggestion
Azure Communication Services can be used within [Azure Government](https://azure.microsoft.com/en-us/global-infrastructure/government/) to provide compliance with US government requirements for cloud services. In addition to enjoying the features and capabilities of Messaging, Voice and Video calling, developers benefit from the following features that are unique to Azure Government:
```

Comment (ProbablePrime) [reply articles/communication-services/concepts/government.md]: We should never abbreviate ACS according to branding.

**Review by ProbablePrime — CHANGES_REQUESTED** [review]
You'll need to edit the toc.yml file in the root of our folder to add a table of contents item for this file. 

Otherwise, mild wording change to navigate around the removal of the abbreviations.

**Comment by hrazi** [comment]
#sign-off

**Pull request merged**